assets/css/nav/_top_nav.scss

```diff
@@ -23,14 +22,26 @@
 }
 
 .c-top-nav__logo-icon {
+  &:not(.c-top-nav__logo-halloween-icon) {
+    margin-left: 1.5rem;
```

Comment:
Oohhhh does this keep 👉  https://github.com/mbta/skate/pull/1314/files#r1823293081 👈  from doing something evil?

Reply:
Yes exaaaactly. I realized I should have included screenshots, so added [those](https://github.com/mbta/skate/pull/1314#issuecomment-2448225695). Basically, pumpkin should be flush to the edge of the navbar when viewport is small, and then look proportional to the left nav when viewport is large. Then I needed to add custom padding on the `Sk8` logo to keep it looking exactly as it is now

Reply:
That's a lot of fiddly work! Well done!